### DISCUSSION TITLE
Where to start contributing

### DISCUSSION THREAD

**Discussion by miwo100** [DaxStudio/DaxStudio#537] [General]
Hi,

I would like to contribute to the project. Do you have any contribution guide ?

Any issue where I could start ? I have a strong development an bi background, but for the beginning I would prefer something easy to get a grasp of the architecture to begin with.

thanks
Michael

**Comment by dgosbell** [comment]
Hi Michael, 

We did not have a contribution guide, so I've just pulled together [this one](https://github.com/DaxStudio/DaxStudio/wiki/Contributing)

However, I've just scanned through our open issues/ideas and I can't see anything that looks like it would be an easy entry point at the moment.

--Darren
